src/components/atoms/Button/index.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/button-has-type */
+
+import React from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+import { ButtonProps } from './types';
+import SpinnerLoader from '~shared/Preloaders/SpinnerLoader';
+
+const displayName = 'Button';
```

Comment:
it's not needed

Reply:
According to the guidelines, we need this
https://hackmd.io/lHcVp96OTBqPCpAnA4ckSQ#DisplayName-and-Component-naming